### ISSUE TITLE
Doc generator: Link to DMTF/Redfish repo mentionned in README_config_files.md ?

### ISSUE THREAD

**Issue by donzef** [DMTF/Redfish-Tools#421]
The [doc-generator/README_config_files.md](https://github.com/DMTF/Redfish-Tools/blob/master/doc-generator/README_config_files.md) asks to clone the DMTF/Redfish repo in order to be able to use the provided json-schema input directory. 

Would it be possible to provide a link to that DMTF/Redfish repo ? So we can use provided examples as-is. 

Thanks.

![image](https://user-images.githubusercontent.com/42031605/181034300-278b70f4-42db-4492-bd20-10ca59c6e67e.png)

**Comment by mraineri** [comment]
Unfortunately that's a private repository only accessible to members of the organization. We shouldn't have any examples in here that reference it. We'll need to clean this up to decouple things.

**Comment by jautor** [comment]
Agreed - that README needs to be updated to remove that reference, and provide more explicit instructions to set up the equivalent.  Essentially, you just need the /json-schema folder from DSP8010 copied locally and referenced in the config file.  The only other thing that would be needed to replicate the contents of DSP2046 / DSP0268 are the "example payload" files.  Those are not currently published, but I will suggest to the group that we include those in future public release bundles.

**Comment by donzef** [comment]
Thanks for the tips. I bumped into DSP2046 file not found errors and, if I remember correctly, I bumped also into a `mockup` or `mockups` file not found.... So, yes, it would be good to have a valid json-schema directory somewhere in the Redfish-Tools repo.

**Comment by dekreia** [comment]
Could you please make DSP2046-examples folder public too? It is required by the "standard_html" sample.
